installation/language/en-GB/joomla.ini

```
@@ -144,6 +144,13 @@ INSTL_LANGUAGES_WARNING_NO_INTERNET="Joomla was unable to connect to the languag
 INSTL_LANGUAGES_WARNING_NO_INTERNET2="Note: You will be able to install languages later using the Joomla Administrator."
 INSTL_LANGUAGES_WARNING_BACK_BUTTON="Return to last installation step"
 
+; Automated Updates Opt Out
+INSTL_AUTOMATED_UPDATES="Automated Updates"
+INSTL_AUTOMATED_UPDATES_DESC="Joomla has a built-in mechanism that will update your Joomla installation automatically.<br /><br /><strong>Please note:</strong> enabling Automated Updates will register your site at the respective service provided by the Joomla! project. The registration allows the Joomla project to access information about your site and server environment, specifically Site Url, PHP version, Database type and version, CMS version and Server OS type and version. This information is only used to improve the service.<br /><br />Alternatively you can disable automated updates using the button below."
```

we dont use xhtml ;)

```suggestion
INSTL_AUTOMATED_UPDATES_DESC="Joomla has a built-in mechanism that will update your Joomla installation automatically.<br><br><strong>Please note:</strong> enabling Automated Updates will register your site at the respective service provided by the Joomla! project. The registration allows the Joomla project to access information about your site and server environment, specifically Site Url, PHP version, Database type and version, CMS version and Server OS type and version. This information is only used to improve the service.<br><br>Alternatively you can disable automated updates using the button below."
```